xml/System/Math.xml

```diff
@@ -124,7 +124,7 @@
       <Docs>
         <param name="value">A number that is greater than or equal to <see cref="F:System.Decimal.MinValue" />, but less than or equal to <see cref="F:System.Decimal.MaxValue" />.</param>
         <summary>Returns the absolute value of a <see cref="T:System.Decimal" /> number.</summary>
-        <returns>A decimal number, x, such that 0 ≤ x ≤ <see cref="F:System.Decimal.MaxValue" />.</returns>
+        <returns>A decimal number, x, such that 0 = x = <see cref="F:System.Decimal.MaxValue" />.</returns>
```

Comment:
@danmoseley there is data loss in this file due to non-ascii characters. Can you please fix them?

I think the easiest thing to do is to revert the file, then copy the few new documentation strings that were truly ported. I noticed the tool (successfully) converted some non-ascii characters to their HTML entity equivalent. (see line 456) in the ported strings.

Apologies for the bug. I am still unsure how to _preserve_ the original file encoding to prevent this, while at the same time handling the elements with System.Xml. I opened a bug to investigate it: https://github.com/carlossanlop/DocsPortingTool/issues/82

Reply:
I'll apply them manually. Much faster/easier.